### PR TITLE
Silicon NTOS

### PR DESCRIPTION
**Pull request by comma**
Needed for ntnet kill PR to work properly since that makes a lot of 'subsystems' unusable without a computer.
Gives synths crew records. AIs get them on spawn (after namepick), borgs get them after they pick a module. This is needed for manifest generation to work withotu special cases for synths
You can now install/remove stock parts into borgs and AIs, currently only used by NTOS. It's done by crowbar (same as components for robots).
Also fixes AIize verb failing if it finds no landmark to move you to.

### CODE REVIEW

**Review by MistakeNot4892 — DISMISSED** [review]

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -617,6 +619,21 @@ var/list/ai_verbs_default = list(
 			user.visible_message("<span class='notice'>\The [user] finishes fastening down \the [src]!</span>")
 			anchored = 1
 			return
+	else if(isCrowbar(W) && user.a_intent != I_HURT)
+		if(!length(stock_parts))
+			to_chat(user, SPAN_WARNING("No parts left to remove"))
+			return
+		var/obj/item/stock_parts/remove = input(user, "Which component do you want to pry out?", "Remove Component") as null|anything in stock_parts
+		if(!remove || !(remove in stock_parts))
+			return
```
Comment (afterthought2) [inline code/modules/mob/living/silicon/ai/ai.dm:628]: Need to re-check CanPhysicallyInteract or similar.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -563,23 +581,28 @@
...
+						I.brute = C.brute_damage
+						I.burn = C.electronics_damage
+
+					removed_item = I
+					if(C.installed == 1)
+						C.uninstall()
+					C.installed = 0
+				else if(istype(remove, /obj/item/stock_parts))
+					stock_parts -= remove
+					removed_item = remove
```
Comment (afterthought2) [inline code/modules/mob/living/silicon/robot/robot.dm:595]: What if it's the matter bin? You haven't cleared the separate ref to it in that case.

**Review by afterthought2 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/ai/ai.dm (hunk 1 above)
Comment (comma) [reply code/modules/mob/living/silicon/ai/ai.dm:628]: that's topic proc, I guess I can check adjacency

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 2 above)
Comment (comma) [reply code/modules/mob/living/silicon/robot/robot.dm:595]: Axed special handling for bin, proc that was using that var now just checks stock parts list for bins and uses their ratings.

File: code/modules/mob/living/silicon/ai/ai.dm (hunk 1 above)
Comment (comma) [reply code/modules/mob/living/silicon/ai/ai.dm:628]: Added adjcancy check

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**